packages/chat_repository_interface/pubspec.yaml

```diff
@@ -0,0 +1,57 @@
+name: chat_repository_interface
+description: "A new Flutter package project."
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: '>=3.4.3 <4.0.0'
+  flutter: ">=1.17.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  
+  rxdart: any
```

Comment:
so we adding rxdart as a dependency to a package just for the local repository interface implementation?